src/table.css.ts

```diff
@@ -0,0 +1,243 @@
+import {createVar, fallbackVar, globalStyle, style, styleVariants} from '@vanilla-extract/css';
+import * as mq from './media-queries.css';
+import {vars as skinVars} from './skins/skin-contract.css';
+import {vars as responsiveLayoutVars} from './responsive-layout.css';
+
+const responsiveLayoutSideMargin = fallbackVar(responsiveLayoutVars.sideMargin, '0px');
+
+const maxHeight = createVar();
+
+export const vars = {maxHeight};
+
+export const table = style({
+    borderCollapse: 'separate',
+    borderSpacing: 0,
+    position: 'relative',
+});
+
+export const fullWidth = style({minWidth: '100%'});
+
+// In mobile, we have 2 rendering modes: horizontal scroll, or collapsed rows. In collapsed rows mode, every row is rendered as a card
+export const collapsedRowsInMobile = style({});
+
+const BOXED_PADDING_Y_DESKTOP = 8;
+
+export const boxed = style({
+    padding: '0 16px',
+    paddingBottom: BOXED_PADDING_Y_DESKTOP,
+    '@media': {
+        [mq.tabletOrSmaller]: {
+            padding: '0 8px',
```

Comment:
This padding left/right is also 16px in the mobile view

Reply:
fixed